plugins/modules/systemd_creds_encrypt.py

```diff
@@ -32,38 +32,43 @@
   name:
     description:
       - The credential name to embed in the encrypted credential data.
+        When using the O(dest) option, you can set this to an empty
+        string ("") to prevent C(systemd-creds encrypt) from using
+        filename from O(dest) as the credential name. Conversely,
+        leave out the O(name) option to let C(systemd-creds encrypt)
+        use the filename from O(dest) as the credential name.
     type: str
-    required: false
   not_after:
     description:
       - The time when the credential shall not be used anymore.
       - Takes a timestamp specification in the format described in V(systemd.time(7\)).
     type: str
-    required: false
   pretty:
     description:
       - Pretty print the output so that it may be pasted directly into a unit file.
     type: bool
-    required: false
     default: false
   secret:
     description:
       - The secret to encrypt.
     type: str
     required: true
+  dest:
+    description:
+      - The destination for the credential file.
+    type: path
+    version_added: 11.2.0
```

Comment:
That version has been released already, must update this to:
```suggestion
    version_added: 11.3.0
```